### PR TITLE
Allow to override channel permissions when creating

### PR DESCRIPTION
**Pull request by pfives**

### CODE REVIEW

File: contracts/makefile [hunk 1]
```diff
@@ -236,8 +236,8 @@ interact-river :;
 # 				 		Base
 # ===========================
 deploy-base :;
-	@$(MAKE) deploy-any-ledger rpc=base hd_path=${HD_PATH} sender=${SENDER_ADDRESS}
-	@$(MAKE) resume-any-ledger rpc=base hd_path=${HD_PATH} sender=${SENDER_ADDRESS} verifier=${BASESCAN_URL} etherscan=${BASESCAN_API_KEY}
+	@$(MAKE) deploy-any-ledger rpc=base hd_path="${HD_PATH}" sender=${SENDER_ADDRESS}
```
Comment (brianathere) [inline contracts/makefile:239]: Was this quoting needed?

File: contracts/src/spaces/facets/channels/Channels.sol [hunk 2]
```diff
@@ -22,6 +23,28 @@ contract Channels is IChannel, ChannelBase, Entitled, Facet {
     _createChannel(channelId, metadata, roleIds);
   }
 
+  function createChannelWithOverridePermissions(
+    bytes32 channelId,
+    string memory metadata,
+    RolePermissions[] memory rolePermissions //how do we pass in a m
```
Comment (brianathere) [inline contracts/src/spaces/facets/channels/Channels.sol:29]: Is this comment truncated?

Comment (brianathere) [reply contracts/src/spaces/facets/channels/Channels.sol:29]: Can this be calldata since you don't mutate it?

File: contracts/makefile (hunk 1 above)
Comment (pfives) [reply contracts/makefile:239]: Yea even if you escape single quotes in hd path in your env_ it breaks running deploy looking for a close '

**Review by brianathere — APPROVED** [review]

**Comment by pfives** [comment]
for some reason merge commit wasn't verified

**Pull request closed without merging**